services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -540,14 +459,15 @@ const RenderAuthCodeFormSection = ({
   );
 };
 
-export const AuthorizationsInvolved = (props) => {
+export const AuthorizationsInvolved = () => {
```

Comment:
initialValues does not need to be passed as a prop when we're using reduxForm. Redux has all that in the state already (both initial & current values, and data about errors, etc) so we can use a selector to get that context _right where we need it_ instead of prop drilling.
Most of the edits here are just taking out props that duplicate redux state.